configs/Samples/AtmSample_nueselec.yaml

```diff
@@ -0,0 +1,108 @@
+SampleName: "nueselec"
+SampleType: "Atm"
+# SelectionCuts:
+#   - KinematicStr: "TrueXPos"
+#     Bounds: [ -310.0, 310.0 ]
+#   - KinematicStr: "TrueYPos"
+#     Bounds: [ -550.0, 550.0 ]
+#   - KinematicStr: "TrueZPos"
+#     Bounds: [ 50.0, 1244.0 ]
+Binning:
+  XVarStr: "TrueNeutrinoEnergy"
+  XVarBins: [0., 0.25, 0.5, 0.75, 1.,  1.25, 1.5, 1.75, 2., 2.25, 2.5, 2.75, 3., 3.25, 3.5, 3.75, 4., 4.25, 4.5, 4.75, 5, 5.5, 6, 6.5, 7, 7.5, 8, 8.5, 9, 9.5, 10, 1e8]
+SampleBools:
+  IsELike: yes
+InputFiles:
+  mtupleprefix: "inputs/Atmospherics/CAFs/atm_hd"
+  mtuplesuffix: "_nueselec.root"
+  splineprefix: ""
+  splinesuffix: ""
+DetID: 24
```

Comment:
Right now, I think the DetID treatment in DUNE is not really used to distinguish samples - should add as issue